site/features.md

```diff
@@ -88,6 +89,22 @@ Containers started in this way that subsequently restart, either by an
 explicit `docker restart` command or by Docker restart policy, are
 re-attached to the weave network by the weave Docker API proxy.
 
+### <a name="plugin"></a>Docker network plugin
+
+Alternatively, you can use weave as a Docker plugin. First you must
+configure the Docker daemon to use a KV store via the
```

Comment:
Docker don't use the term "KV store"; they do use "KV service" once.  Maybe just "configure the Docker daemon to use a cluster store as described here"